astropy/modeling/tests/example_models.py

```diff
@@ -194,11 +194,28 @@
         'requires_scipy': True,
         'x_lim': [0,10],
         'log_fit': True
-    }
+    },
+
+    Voigt1D: {
+        'parameters': [0, 1, 0.5, 0.9],
+        'x_values': [0, 2, 4, 8, 10],
+        'y_values': [0.71638049, 0.0484249, 0.01496101, 0.00424706, 0.00279052],
+        'x_lim': [-3, 3]
+     },
+
+    Sersic1D: {
+        'parameters': [1, 20, 4],
+        'x_values': [0.1, 1, 10, 100],
+        'y_values': [2.78629391e+02, 5.69791430e+01, 3.38788244e+00,
+                     2.23941982e-02],
+        'requires_scipy': True,
+        'x_lim': [0,10],
+        'log_fit': True
+    },
```

Comment:
Oops, this still hung around from the botched rebase (it's a duplicate of the entry just above `Voigt1D`).  You can just remove this, and we'll get everything squashed and cleaned up before merging.